source/keymanweb.js

```diff
@@ -3542,16 +3574,97 @@
     {
       if(document.attachEvent)
         document.attachEvent('onfocusin', keymanweb._IEFocusIn);
-      else if(document.addEventListener)
-        document.addEventListener('DOMNodeInserted', keymanweb._DOMNodeInserted, true);
     }
    
     // Restore and reload the currently selected keyboard 
     keymanweb.restoreCurrentKeyboard(); 
 
+      /* Setup of handlers for dynamically-added and (eventually) dynamically-removed elements.
+       * Reference: https://developer.mozilla.org/en/docs/Web/API/MutationObserver
+       * 
+       * We place it here so that it loads after most of the other UI loads, reducing the MutationObserver's overhead.
+       */
+
+      var observationTarget = document.querySelector('body');
+      keymanweb.mutationObserver = new MutationObserver(function(mutations) 
+        {
+          var dirtyFlag = false; // Notes if we need to recompute our .sortedInputs array.
+
+          for(var i=0; i < mutations.length; i++)
```

Comment:
We probably need a style guide but for now let's keep `{` and `}` consistent:
````
if(x) {
  ...
} else {
  ...
}
````

And `var i = 0;` not `var i=0;`   :)